README.md

```diff
@@ -6,30 +6,24 @@ A script that uses sphinx to develop documentation for ApeWorX plugins.
 
 - [python3](https://www.python.org/downloads) version 3.9 up to 3.12.
 
-## Installation
-
-### via `pip`
+## Quick Usage
 
-You can install the latest release via [`pip`](https://pypi.org/project/pip/):
+To use this to build the documentation in an Ape plugin, add this to the `docs.yaml`
 
 ```bash
-pip install docs-build
-```
+        - name: Clone ape-docs-build
```

Comment:
idea: Perhaps we should make a custom github actions so users can do:

```
uses: ApeWorX/ape-docs-build
```

Reply:
I tried working on making it a custom github action but I think I would have to rework some of it because of how I currently have it setup.

Reply:
I can try to help